app/components/bs-tooltip/component.js

```diff
@@ -22,6 +22,6 @@ export default Ember.Component.extend(BootstrapComponentOptions, {
     var options = this.getBootstrapOptions();
     var tooltip = this.$(':first-child').tooltip(options);
 
-    if(options.trigger = 'immediate') { tooltip.tooltip('show'); }
+    if(options.trigger === 'immediate') { tooltip.tooltip('show'); }
```

Comment:
Just a style nit, but using the multi-line version of `if` is preferred, since you can then more easily add a debugger to a line. For example, adding a debugger to the `tooltip` function here would be tricky.